app/lib/modules/llm/whitelist.ts

```diff
@@ -0,0 +1,60 @@
+import type { ModelInfo } from './types';
+
+/**
+ * 화이트리스트 항목 인터페이스
+ * label: 사용자에게 표시될 이름
+ * providerName: 실제 프로바이더 이름
+ * modelName: 실제 모델 이름
+ */
+export interface WhitelistItem {
+  label: string;
+  providerName: string;
+  modelName: string;
+}
+
+/**
+ * 화이트리스트 목록
+ * 사용자에게 제공할 프로바이더와 모델 조합을 정의합니다.
+ */
+export const MODEL_WHITELIST: WhitelistItem[] = [
+  {
+    label: 'Claude 3.7 Sonnet (OpenRouter)',
+    providerName: 'OpenRouter',
+    modelName: 'anthropic/claude-3.7-sonnet',
+  },
+  {
+    label: 'Claude 3.7 Sonnet (Anthropic)',
+    providerName: 'Anthropic',
+    modelName: 'claude-3-7-sonnet-20250219',
+  },
+];
```

Comment:
If we plan to allow a small number of models, it could be better to set the model key to be allowed in an environment variable or something similar.

Reply:
I also thought it would be better to call it from .env, so I created an issue, but I realized that this project is not easy to process calling the environment variables, so I'm planning to proceed as it is for now due to the rather complicated modifications.